kafka/producer/kafka.py

```diff
@@ -353,7 +354,7 @@ def __init__(self, **configs):
         if self.config['compression_type'] == 'lz4':
```

Comment:
Since compression_type is a producer configuration, we could put the check for compression libraries import in `__init__` here